core/src/main/java/io/camunda/connector/impl/OutboundConnectorConfiguration.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.connector.impl;
```

Comment:
Should we move this to the `outbound` package?
```suggestion
package io.camunda.connector.impl.outbound;
```

Reply:
Accounted for via https://github.com/camunda/connector-sdk/pull/104/commits/83512651a0c71982c97fba17c3d50e283958d53d.

Reply:
I might be overlooking something here, but shouldn't we also move the file accordingly to the package path? 🤔

Reply:
Not sure at all what happened here, but it should be in the right place now :1234: